tutorials/pyMBE_tutorial.ipynb

```diff
@@ -1625,14 +2374,122 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 36,
```

Comment:
Please clear output cells before committing notebook changes :)

Reply:
ups! I thought I did clean it up, but probably I forgot to save it afterwards...